### ISSUE TITLE
Inventory will not load

### ISSUE THREAD

**Issue by paigegriffith** [CasperTech/METAbolt#23]
The inventory will not load for this - how do we fix these. Currently we are using Metabolt 2

**Comment by Kayamarazaki** [comment]
same issue on V 0.9.71. 
it gets stuck on loading, if you refresh one time you get an error.. inventorynull I believe.. and the other refresh you get no error yet it does not load. 
Tested with sending a landmark whilst logged into sl and metabolt, you can accept but does not show up in inventory =/

**Comment by Cherona** [comment]
I am using the most up-to-date version and my inventory won't load either.  This seemed to happen on Tuesday when SL did its rolling restarts.  Coincidence maybe, but as some of my worn items become detached when using with app, I cannot reattach them as my inventory will not load at all.

**Comment by KindaFallen** [comment]
It has been going on for awhile before last tues roll out, he gave me an answer yesterday to my original question about it, about something being capped, yada yada geek talk, and they were working on a new version of metabolt that would handle inventory again.

**Comment by Kayamarazaki** [comment]
sure hope it'll be soon, the metabolt note plugins are lifesaving for running a sl business ;p

**Comment by Macsima** [comment]
same here, Inventory does not load. I only got a popup named "exception report"
with the message "Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt."
(in english: Object reference not set to an object instance.)

![metabolt exception report](https://cloud.githubusercontent.com/assets/11876427/7075125/e4c45de8-deff-11e4-8e37-0457137262a6.jpg)

**Comment by GenoLupindo** [comment]
Found a work around just move all note cards and LM's out of the folders to the main inventory window and you can use them.

**Comment by Kayamarazaki** [comment]
nice might try that for my landmark!

**Comment by Macsima** [comment]
yes, great idea, that helps a lot!

**Comment by paigegriffith** [comment]
@geno if the inventory won't load at all how am I supposed to move the LM??
We can't see ANYTHING in the inventory. Even if we try to create new
landmark it just goes into the default LM folder created for SL and it
won't load. :(

**Comment by Macsima** [comment]
Heya Paige!
You CAN see the first level of folders, but you cannot open them.
I went online with firestorm and the SL Viewer and there is all ok. So there I could move the needed landmark to that first level.
Back in Metabolt I could see and use those LM's.

**Comment by Kayamarazaki** [comment]
just saying, akamai is dressed when I log her into metabolt and I use my main to perv her >3

**Comment by Macsima** [comment]
My Alt is only dressed  when I log in with a  'normal' viewer

**Comment by Kayamarazaki** [comment]
hmmmm mine's dressed whilst logged in with metabolt Oo 
even her AO works and all

**Comment by Cherona** [comment]
Random pieces of my avatar disappear.  This is what I'm told by people who are in-world while I am on MetaBolt.  For instance, everything is loaded but my mesh body, so they see a floating head with clothes. Since the inventory won't load I can't even go in an reattach it, which I could in the past.  And I only noticed this happening on Tuesday; whether it was happening before that or not I have no idea.  This seems like a pretty significant bug, as I can't accept landmarks or notecards and use them. :/

**Comment by GenoLupindo** [comment]
To move items in your Inv log in with a normal SL Viewer

**Comment by Kayamarazaki** [comment]
Don't bother, there's not going to be a fix nor updates etc. 
best is to wait and see what the new viewer is going to be (if there is any coming out)
use your main viewer to get things to work around this bug for the time being but prepare to say bye bye metabolt

**Comment by Macsima** [comment]
Ok, I think, nobody will fix that anymore :(
I just read that note:
CasperTech withdraws from METAbolt project

It is with regret that we must announce end of support of METAbolt from CasperTech.

We have been asked to cease using the METAbolt brand with immediate effect, due to a political disagreement with the original creator.

With no maintainer, it appears that METAbolt will no longer be developed.

We would like to thank the community for their overwhelming support during the project's time with us.

CasperTech plan to release an alternative client shortly, under a different name.

**Comment by ajwsert** [comment]
Well, I'm not sure if it's already a definitive decision or if something can change and they 
step back and continue.. (I hope the latter) but for now a thing I could suggest to investigate the issue is using a previous version. Have someone tested to see if it happens in older versions? If so then the issue could be on MB, otherwise the issue lies on the way as MB communicates with SL servers maybe.

**Comment by ajwsert** [comment]
Uh oh, I think I found a hint about what it can be... http://wiki.secondlife.com/wiki/Release_Notes/Second_Life_Server/#15.03.30.300351

"Code cleanup around inventory entailing removal of deprecated protocol path"

**Comment by Kayamarazaki** [comment]
it's a decisive thing hun... caspertech has decided to quit... they aint gonna come back like: nah just kidding ;p

**Comment by TomMettam** [comment]
It wasn't primarily our decision, the creator asked us to cease.

**Comment by ajwsert** [comment]
Yes, I know that. Well, what I can do then.. I just had hopes one or both sides would change their minds... I just saw it as a misunderstanding so it could be solvable... (One can hope) :/

**Comment by scnupage** [comment]
He's asked you to cease but won't develop it himself?? How very disappointing!

**Comment by ajwsert** [comment]
Yeah, that was one of the reasons that CasperTech took the project on first place: the original author couldn't maintain the project anymore. (late reply, I know, but I saw it only now :P)
